pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -91,7 +91,7 @@
 , withSsh ? withHeadlessDeps # SFTP protocol
 , withSvg ? withFullDeps # SVG protocol
 , withSvtav1 ? withHeadlessDeps && !stdenv.isAarch64 && !stdenv.hostPlatform.isMinGW # AV1 encoder/decoder (focused on speed and correctness)
-, withTensorflow ? false # Tensorflow dnn backend support
+, withTensorflow ? false # Tensorflow dnn backend support # Increases closure size by ~390 MiB
```

Comment:
If it does indeed eval without allowInsecure now and "only" adds 390MiB, we could think about enabling it.

I'd like to resolve https://github.com/NixOS/nixpkgs/issues/271863 beforehand though.